<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>4</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Моделировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_request_count">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество заявок:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>240</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вероятность повторной
обработки заявки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_reenter_probability">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Метод моделирования:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deltat">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Δt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>217</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Количество повторно
обработанных заявок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_reentered_count">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>206</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальная длина
очереди</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_max_queue_size">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_servicemachine_m">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">μ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_generator_a">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>12</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_generator_b">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры генератора(равномерное распределение):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры обработчика(нормальное распределение):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">D</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_servicemachine_d">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>193</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_reentered_count_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_deltat_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Событийный</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_max_queue_size_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_deltat_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Δt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deltat">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
